crates/wallet/src/wallet/tx_builder.rs

```diff
@@ -763,16 +766,33 @@ impl fmt::Display for AddForeignUtxoError {
 #[cfg(feature = "std")]
 impl std::error::Error for AddForeignUtxoError {}
 
+type TxSort<T> = dyn Fn(&T, &T) -> std::cmp::Ordering;
```

Comment:
🔧 Need to change the `Ordering` type to be compatible with no-std (and WASM).

```suggestion
type TxSort<T> = dyn Fn(&T, &T) -> core::cmp::Ordering;
```